pol-core/bscript/object.cpp

```diff
@@ -2418,6 +2479,20 @@ BObjectImp* BFunctionRef::call_method_id( const int id, Executor& ex, bool /*for
                                       ex.numParams() ) );
     }
   }
+  case MTH_NEW:
+  {
+    if ( variadic_ )
+    {
+      return new BError( fmt::format( "Invalid argument count: expected {}+, got {}",
+                                      static_cast<int>( num_params_ ) - 2 /*remove this*/,
```

Comment:
no need for cast, and probably quote "this" so it's not "remove this" (which may imply "remove this" the code)